codechecker_lib/analyzers/result_handler_plist_to_db.py

```diff
@@ -164,16 +165,19 @@ def handle_results(self):
 
             _, source_file_name = ntpath.split(self.analyzed_source_file)
 
+            if LoggerFactory.get_log_level() == logger.DEBUG:
+                analyzer_cmd = ''
```

Comment:
We set analyzer_cmd to empty string only when the logger is in debug mode? Did you mean the opposite way?